Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -909,10 +910,13 @@ def main() -> None:
     :rtype:
     """
     # get pack version
-    response = demisto.internalHttpRequest("GET", "/contentpacks/metadata/installed")
-    packs = json.loads(response["body"])
+    if StrictVersion(demisto.demistoVersion()['version']) >= StrictVersion("6.1.0"):
+        response = demisto.internalHttpRequest("GET", "/contentpacks/metadata/installed")
+        packs = json.loads(response["body"])
+    else:
+        packs = []
 
-    pack_version = "1.1.0"
+    pack_version = "1.1.1"
```

Comment:
Every time we update the pack (and bump the version) we will need to increase the pack_version in the code?

Reply:
This is a default value for the pre-6.1 versions since there doesn't appear to be a way to dynamically retrieve the pack version prior to 6.1.  Ideally, it is updated with any update to the pack, but it is non-critical if that doesn't happen.  I don't expect the Palo team to update this value if they push updates.  We can handle it on any updates that we contribute.